core/src/main/java/io/grpc/Attributes.java

```diff
@@ -31,9 +31,13 @@
 
 package io.grpc;
 
+import static com.google.common.collect.Maps.newHashMap;
```

Comment:
We need to avoid Guava collect. We have an armistice with some Android teams that we can use Guava if we don't use Guava collect.

There are actually places where we are using it now that have snuck in, but I was going to be removing them.